src/client/datascience/jupyter/liveshare/hostJupyterNotebook.ts

```diff
@@ -27,29 +16,16 @@ import {
     InterruptResult
 } from '../../types';
 import { JupyterNotebookBase } from '../jupyterNotebook';
-import { LiveShareParticipantHost } from './liveShareParticipantMixin';
-import { ResponseQueue } from './responseQueue';
-import { IRoleBasedObject } from './roleBasedFactory';
-import { IExecuteObservableResponse, IResponseMapping, IServerResponse, ServerResponseType } from './types';
-
-// eslint-disable-next-line @typescript-eslint/no-require-imports
-import cloneDeep = require('lodash/cloneDeep');
 import { IFileSystem } from '../../../common/platform/types';
 import { IPythonExecutionFactory } from '../../../common/process/types';
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
-export class HostJupyterNotebook
-    extends LiveShareParticipantHost(JupyterNotebookBase, LiveShare.JupyterNotebookSharedService)
-    implements IRoleBasedObject, INotebook {
-    private catchupResponses: ResponseQueue = new ResponseQueue();
-    private localResponses: ResponseQueue = new ResponseQueue();
+export class HostJupyterNotebook extends JupyterNotebookBase implements INotebook {
```

Comment:
Could probably get rid of the 'host' classes and the 'base' class. We just need one now. ALthough you could do that after this.

Reply:
Yes that's the plan, this way it's easier to see the differences. 
Else renaming, moving will add noise to the pr and make it more difficult to review.
But all this feedback is welcome